lib/eslint-rails/runner.rb

```diff
@@ -12,52 +14,69 @@ def run
       warnings = assets.map do |asset|
         generate_warnings(asset).tap { |warnings| output_progress(warnings) }
       end
-      
+
       puts
-      
+
       warnings.flatten
     end
-    
+
     private
-    
+
     def normalize_infile(file)
       file = file.to_s.gsub(/^app\/assets\/javascripts\//, '') # Remove beginning of asset path
       file = Pathname.new("#{Dir.pwd}/app/assets/javascripts/#{file}") # Ensure path is absolute
       file = Pathname.new("#{file}.js") if !file.directory? && file.extname.empty? # Make sure it has an extension
       file
     end
-    
+
     def assets
-      all_js_assets = Rails.application.assets.each_file.to_a.select { |pn| pn.extname == '.js' }
-      
+      all_js_assets = Rails.application.assets.each_file.to_a.select { |pn| JAVASCRIPT_EXTENSIONS.include?(pn.extname) }
+
       assets = all_js_assets.select{|a| is_descendant?(@file, a)}
 
       assets.reject{|a| a.to_s =~ /eslint.js|vendor|gems|min.js|editorial/ }
     end
-    
+
     def eslint_js
       @eslint_js ||= Rails.application.assets['eslint'].to_s
     end
-    
+
+    def eslint_plugin_js
+      @eslint_plugin_js ||= begin
+        plugins.map do |plugin_name|
+          Rails.application.assets["plugins/eslint-plugin-#{plugin_name}"].to_s
+        end.join('\n')
+      end
+    end
+
+    def plugins
+      JSON.parse(Config.read)['plugins'] || []
+    end
+
+    def react_plugin_js
```

Comment:
Good call. I think I was using this to test things before I made eslint_plugin_js